hammerblade/torch/kernel/kernel_eye.cpp

```diff
@@ -0,0 +1,31 @@
+#include <kernel_common.hpp>
+
+
+extern "C" {
+
+  __attribute__ ((noinline)) int tensorlib_eye(
+    hb_tensor_t* output, long* n, long* m) {
+    auto y = HBTensor<float>(output);
+    auto N = *n;
```

Comment:
I general we should not abuse auto. The type `long` is the same number of characters as `auto`. So this should probably be:
```
HBTensor<float> y(output);
long N = *n;
```
The above is more explicit, more compact, and more readable. Just a thought :)
```

Reply:
Just made the change